.github/workflows/build-and-publish.yml

```
@@ -54,212 +78,201 @@ jobs:
-          # for AssemblyVersion/FileVersion
-          CLEAN_VERSION=$(echo "$VERSION" | sed 's/-.*$//')
-          echo "clean_version=$CLEAN_VERSION" >> $GITHUB_OUTPUT
-          echo "Clean version for assemblies: $CLEAN_VERSION"
+          # Function to increment version
+          increment_version() {
+            local version=$1
+            IFS='.' read -r major minor patch <<< "$version"
+            patch=$((patch + 1))
+            echo "${major}.${minor}.${patch}"
+          }
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)